k8s/distributed-chroma/templates/compaction-service.yaml

```diff
@@ -29,28 +29,42 @@ spec:
         member-type: compaction-service
     spec:
       serviceAccountName: compaction-service-serviceaccount
-      {{if .Values.compactionService.configuration}}
       volumes:
+        {{if .Values.compactionService.configuration}}
         - name: compaction-service-config
           configMap:
             name: compaction-service-config
-      {{ end }}
+        {{ end }}
+        {{if .Values.compactionService.cache}}
```

Comment:
nit: cache_path?

Reply:
I am using it as a object with two values below it, one for `hostPath` and the other for `mountPath` so I was thinking this should just be `cache`. Or were you suggesting something else?